monitoring_service.rst

```diff
@@ -219,18 +219,18 @@ Example Data: Monitor Request
 
     {
       "onchain_balance_proof": {
-          "signature_prefix": tbd,
-          "message_length": 212,
-          "token_network_address": tbd,
-          "chain_id": tbd,
+          "signature_prefix": "13",
+          "message_length": "212",
```

Comment:
Is this a string?

Reply:
Seems like it has to be a string. Can you make this clearer in the **Note** box:

Length of the actual message to be signed in decimal representation (not null-terminated).

to

String of the length of the actual message to be signed in decimal representation (not null-terminated).